LuaMenu/widgets/chobby/components/configuration.lua

```diff
@@ -545,6 +554,9 @@ function Configuration:SetConfigValue(key, value)
 		lobby.useSpringRestart = value
 		localLobby.useSpringRestart = value
 	end
+	if key == "disableColorChoosing" then
```

Comment:
disableColorChoosing should exist in luamenu saved widget data, not any form of springsettings.

Reply:
And why does it get sent to localLobby? The lobby protocols would ideally deal with things directly related to the server (or making a fake server in the case of singleplayer). Setting teamcolour is firmly in the domain of the lobby state and UI, hence should be in configuration.